### ISSUE TITLE
Is it possible to disable "Grant Access" button?

### ISSUE THREAD

**Issue by marbuser** [xfrocks/bdApi#127]
I'm curious if it's possible for us to disable the "grant access" stuff? I'm only going to allow this api to be used for my own apps and not public, and I want to avoid my users having to click grant. I would ideally like to make it so that if they are logged in it just auto redirects them back with auth token instead of them having to click button first.

Is this possible? Thanks.

**Comment by daohoangson** [comment]
Yes, you can edit the client from AdminCP and auto grant some scopes. If the requested scopes are auto granted, user will be redirected immediately. You should only enable this features for trusted client though, because user won't be able to deny access this way.
